lib/api/src/js/externals/table.rs

```diff
@@ -11,9 +11,7 @@ pub struct Table {
     pub(crate) handle: VMTable,
 }
 
-// Table can't be Send in js because it dosen't support `structuredClone`
-// https://developer.mozilla.org/en-US/docs/Web/API/structuredClone
-// unsafe impl Send for Table {}
```

Comment:
Can we keep a comment on all `assert_not_implemented` usage points so future readers know what's going on here?